### ISSUE TITLE
Provide RGB Only Model

### ISSUE THREAD

**Issue by MSLacerda** [mchengny/RWF2000-Video-Database-for-Violence-Detection#14]
Greetings,

First, I want to thank you for sharing this incredible work.

And I wanna ask if it is possible to you to share the RGB Only model? I'm having problem to reproduce the training step of this one.

I'm running the training on a colab enviroment and loss keeps on the same value: (0.69) in all the epochs.

Thanks!

**Comment by MSLacerda** [comment]
Hi, I could solve it and reproduce the RGB model.

**Comment by WENDGOUNDI** [comment]
Hello @MSLacerda, can you state how you have run the RGB Only model? Thank you

**Comment by freak073** [comment]
> Hi, I could solve it and reproduce the RGB model.

how did u do it? can u tell me
